src/Diagrams/Swimunit/Dotmatrix.hs

```diff
@@ -0,0 +1,843 @@
+{-# LANGUAGE NoMonomorphismRestriction #-}
+
+module Diagrams.Swimunit.Dotmatrix
```

Comment:
Hi Chris,
concerning the TwoD package I can not tell. But personally I would like 
to keep Dotmatrix as the emphasis is neither on dot nor on matrix, its 
the combination of both and therefore the two nouns should not be 
separated by a Camel.
But: If community naming rules are different I will comply to these.

Regards
     Wolfram





On 18.09.2015 20:13, Chris wrote:
>
> In src/Diagrams/Swimunit/Dotmatrix.hs 
> <https://github.com/diagrams/diagrams-contrib/pull/55#discussion_r39884396>:
>
> > @@ -0,0 +1,843 @@
> > +{-# LANGUAGE NoMonomorphismRestriction #-}
> > +
> > +module Diagrams.Swimunit.Dotmatrix
>
> Would |Diagrams.TwoD.DotMatrix| be more consistent with other modules 
> names?
>
> Also remember to add this to |diagrams-contrib.cabal|.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/diagrams/diagrams-contrib/pull/55/files#r39884396>.
>